open_city_profile/__init__.py

```diff
@@ -0,0 +1 @@
+__version__ = "1.13.1"
```

Comment:
I guess release-please automatically bumps this?

Reply:
yes, release-please should bump this.